src/isValidRequest.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+'use strict';
+
+const allowedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function isValidRequest(text, toCase) {
+  const statusText = {
+    errors: [],
+  };
+
+  if (!text) {
+    statusText.errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
You can create an object with all your default error messages and use them as reference